try {
	eat();
}
catch (Exception up) {
	throw up;
}